### PR TITLE
Compile and install recover utility for nethack crashes

### PR DESCRIPTION
**Pull request by rakarnik**

### CODE REVIEW

File: nethack.rb [hunk 1]
```diff
@@ -67,7 +67,14 @@ def install
       system "make"
     end
 
+	# Make utilities
```
Comment (mistydemeo) [inline nethack.rb]: Indent using spaces rather than tabs, please!

File: nethack.rb [hunk 2]
```diff
@@ -67,7 +67,14 @@ def install
...
+	cd "util" do
+      system "make recover"
```
Comment (dunn) [inline nethack.rb]: Use spaces rather than tabs for line 71, and make line 72 read `system "make", "recover"`

File: nethack.rb [hunk 3]
```diff
@@ -67,7 +67,13 @@ def install
       system "make"
     end
 
+    # Make utilities
+    cd "util" do
+      system "make", "recover"
+    end
+
     bin.install "src/nethack"
+	bin.install "util/recover"
```
Comment (mistydemeo) [inline nethack.rb]: Spaces here too, please!

**Comment by rakarnik** [comment]
bumping... anything more you need from me?

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by mistydemeo** [comment]
This is included in the latest nethack! Sorry for not acting sooner.

**Pull request closed without merging**